cli/Dockerfile-bin

```diff
@@ -1,8 +1,9 @@
 ## compile binaries
-FROM gcr.io/runconduit/go-deps:a6e8221d as golang
+FROM gcr.io/runconduit/go-deps:ff0dc047 as golang
 ARG CONDUIT_VERSION
 WORKDIR /go/src/github.com/runconduit/conduit
 COPY cli cli
+COPY controller/k8s controller/k8s
```

Comment:
is this for the parsing utilities? does the controller actually need the parsing stuff? can they be moved to the cli package?  I'm looking to minimize docker cache invalidation for the CLI...

Reply:
Unfortunately it's `controller/api/public/grpc_server.go` depending on `controller/k8s.Lister`. Since `Dockerfile-bin` already pulls in `controller/api`, it now must pull in `controller/k8s`:
https://github.com/runconduit/conduit/blob/38cc23e67e521eff0e491cb04263129040fb3d1d/controller/api/public/grpc_server.go#L14